### PR TITLE
Use two letter country

### LINKED ISSUE #265: Disconnected: Country name must be a 2 character country code
The instructions doesn't say anything about the country code should be a two character country code (https://github.com/openshift-kni/baremetal-deploy/tree/master/ansible-ipi-install#disconnected-registry), actually the example says `cert_country=MyCountry` (which doesn't work)

Also, I find the disconnected instructions confusing. For instance, the "Creating a New Disconnected Registry" IMHO should include the `[registry_host]` as well:

```
[registry_host]
xxx

[registry_host:vars]
cert_country=ES
cert_state=MyState
cert_locality=MyCity
cert_organization=MyCompany
cert_organizational_unit=MyDepartment
```

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Replaces MyCountry example to have 'ES' and adds a comment so that users know that must be replaced and still provides a valid default for installation to continue

Fixes #265

## Type of change

Please select the appropiate options:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [x] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [x] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [x] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: ansible-ipi-install/README.md [hunk 1]
```diff
@@ -321,7 +321,7 @@ provisioner.example.com
 # The following cert_* variables are needed to create the certificates
 #   when creating a disconnected registry. They are not needed to use
 #   an existing disconnected registry.
-# cert_country=MyCountry
+# cert_country=ES #it must be two letters country
```
Comment (rlopez133) [inline ansible-ipi-install/README.md:324]: can we default it to US? (don't hate me lol)

File: ansible-ipi-install/README.md [hunk 2]
```diff
@@ -422,7 +422,7 @@ The certificate information used to generate the host certificate must be define
 
 ```ini
 [registry_host:vars]
-cert_country=MyCountry
+cert_country=ES # two letters country
```
Comment (rlopez133) [inline ansible-ipi-install/README.md:425]: same as above

File: ansible-ipi-install/README.md (hunk 1 above)
Comment (iranzo) [reply ansible-ipi-install/README.md:324]: Yes, it is already where US are, that is ES ;-)

Comment (iranzo) [reply ansible-ipi-install/README.md:324]: Will update it tomorrow

**Comment by rlopez133** [comment]
@iranzo - can we add the fix also to the inventory/hosts.sample file?

**Comment by iranzo** [comment]
> @iranzo - can we add the fix also to the inventory/hosts.sample file?

Done!

**Pull request merged**